### DISCUSSION TITLE
creating my own character

### DISCUSSION THREAD

**Discussion by Piggymaster1222** [JordanSantiagoYT/FNF-JS-Engine#830] [Q&A] [answered]
I was wanting to make a chart for Hjoim remake and I made my own character sprite but I seem to be having trouble. is there any way I can fix this issue? I do have the png sheet thankfully
![hjoim](https://github.com/user-attachments/assets/71c4a25b-2a36-488b-b70c-385b7c7a853f)

**Comment by JordanSantiagoYT** [comment]
do you have the xml?

**Reply by imavie** [reply]
yeah does he have a XML file

**Reply by Piggymaster1222** [reply]
> do you have the xml?

Yes I do. But everytime I try loading the image and xml, the animations just say "ERROR!"

**Reply by Piggymaster1222** [reply]
> > do you have the xml?
> 
> Yes I do. But everytime I try loading the image and xml, the animations just say "ERROR!"

![Screenshot 2025-01-01 13-10-13](https://github.com/user-attachments/assets/5920fad8-0c3b-456a-a782-e04834f27113)

**Reply by imavie** [reply]
press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there

**Reply by Piggymaster1222** [reply]
> press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there

Ok

**Reply by Piggymaster1222** [reply]
> press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there

im still struggling with it. is there any more advice?

**Reply by Piggymaster1222** [reply]
> > press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there
> 
> im still struggling with it. is there any more advice?

and it still shows the error thing

**Reply by Piggymaster1222** [reply]
> > > press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there
> > 
> > 
> > im still struggling with it. is there any more advice?
> 
> and it still shows the error thing

![Screenshot 2025-01-01 15-19-38](https://github.com/user-attachments/assets/aca95f77-c4d7-4d20-ba22-000b7e6109d5)

**Reply by imavie** [reply]
go to the animations tab and try to understand smth

**Reply by Piggymaster1222** [reply]
> go to the animations tab and try to understand smth

im new to making my own sprite :.(

**Reply by Piggymaster1222** [reply]
im sorry :(

**Reply by moxie-coder** [reply]
> > > > press "Load Template" and put your sprite again, and go to the animations tab, you can figure it out there
> > > 
> > > 
> > > im still struggling with it. is there any more advice?
> > 
> > 
> > and it still shows the error thing
> 
> ![Screenshot 2025-01-01 15-19-38](https://private-user-images.githubusercontent.com/179584620/399586026-aca95f77-c4d7-4d20-ba22-000b7e6109d5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU3NjM1NTgsIm5iZiI6MTczNTc2MzI1OCwicGF0aCI6Ii8xNzk1ODQ2MjAvMzk5NTg2MDI2LWFjYTk1Zjc3LWM0ZDctNGQyMC1iYTIyLTAwMGI3ZTYxMDlkNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwMVQyMDI3MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MmU3YWE3YjI2OWUwZDIzZWNjMTIwNGZjNzZjNjdkNjY3YzViZDczNDU3YmYyYTNiYjBjNzBiMDc1MjQ3ZDEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.XcxLe-Prq5kSDDBr88zr1TAHCI5ouSnKuuLgwZ6PvsY)

you need to set up your animations for your character, which you can read about here https://github.com/ShadowMario/FNF-PsychEngine/wiki/Creating-a-Character

**Reply by Piggymaster1222** [reply]
I used something different to make a sprite tho

**Reply by Piggymaster1222** [reply]
I used this 
![image](https://github.com/user-attachments/assets/115ba783-d2b1-4575-99c1-09a9d7d07db0)

**Reply by moxie-coder** [reply, marked as answer]
> I used this ![image](https://private-user-images.githubusercontent.com/179584620/399586524-115ba783-d2b1-4575-99c1-09a9d7d07db0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU3NjM5NTUsIm5iZiI6MTczNTc2MzY1NSwicGF0aCI6Ii8xNzk1ODQ2MjAvMzk5NTg2NTI0LTExNWJhNzgzLWQyYjEtNDU3NS05OWMxLTA5YTlkN2QwN2RiMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEwMVQyMDM0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNjIyODhhYTU4MjVjOTAyMTQ1ODVjN2FlMjYzZDEwYzlkZmMwNzc1YTE5OWVhMmYzMDk4OWRlNTI2NDE4NDhjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.0s-CXrYYq9aB4Y6hPRRF2OepjzIixU4qBpN2sfvOiPo)

that doesn’t matter, sprite sheet creators are fine you just have to make animations for your character based on the names in the xml, which again you can read about here https://github.com/ShadowMario/FNF-PsychEngine/wiki/Creating-a-Character

**Reply by Piggymaster1222** [reply]
oh...ok...like I said, I am making my very first sprite :/
